model-sync-gradle-test/ci.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+(
+  cd "$(dirname "$0")"
+
+  (
+    cd graph-lang-api
+    ./gradlew publishToMavenLocal --console=plain
+  )
+
+  ./gradlew assemble --console=plain
+
+  if [ "${CI}" != "true" ]; then
+    trap cleanup INT TERM EXIT
+    cleanup () {
+      kill "${MODEL_SERVER_PID}"
+      exit
+    }
+  fi
+
+  ./gradlew runModelServer --console=plain &
+  MODEL_SERVER_PID=$!
+  sleep 5
+
+  #CI needs more time
+  if [ "${CI}" = "true" ]; then
+    sleep 10
+  fi
```

Comment:
You can make this more reliable by replacing it with [wait-for-it](https://github.com/vishnubob/wait-for-it). That waits for the target port to become available.